### DISCUSSION TITLE
Displaying "Items 1–n of m" for a listing

### DISCUSSION THREAD

**Discussion by tim-at-topos** [quarto-dev/quarto-cli#8668] [Q&A]
### Description

I have a listing in my Quarto website that uses a custom `.ejs` template. I'd like it to display e.g. "Showing items 6–10 of 23" along with the pagination, so that the viewer can tell how many total items there are. Is there currently a way of doing this?

I can imagine maybe using the `items` list in the `.ejs` file and calculating its length to get the total number, but is there a way to get the values of the currently displayed items in the listing from Quarto?

**Comment by mcanouil** [comment]
It might be possible.
You should look into the EJS documentation, see <https://ejs.co/>.
